src/BundleSource.ts

```diff
@@ -225,7 +225,8 @@ ${file.headerContent ? file.headerContent.join("\n") : ""}`);
             this.concat.add(null, "(FuseBox)");
         }
         if (this.context.sourceMapsProject || this.context.sourceMapsVendor) {
-            this.concat.add(null, `//# sourceMappingURL=${this.context.bundle.name}.js.map`);
+            let sourceName = /[^\/]*$/.exec(this.context.bundle.name)[0];
```

Comment:
As I described in the issue, we need to keep the file name out of the bundle name :
`/[^\/]*$/.exec('path/to/bundle')[0]  ==>  'bundle'